src/centrality.rs

```diff
@@ -326,3 +326,119 @@ pub fn digraph_eigenvector_centrality(
         ))),
     }
 }
+
+/// Compute the edge betweenness centrality of all edges in a :class:`~PyGraph`.
```

Comment:
The edge and vertex centrality functions should be next to each other in this file, as they are more closely related to each other than to the eigenvector centrality... unless there is another good reason for the current order.